neurons/validators/validator.py

```diff
@@ -172,7 +173,20 @@ def get_reward(self, response: DiscoveryOutput, ip_per_hotkey=None, run_id_per_h
         )
 
         if self.validator_config.is_grace_period and response.version == 5:
-            score = max(score, self.validator_config.grace_threshold)
+            covered_blocks = last_block_height - start_block_height
+            ideal_score = self.scorer.calculate_score(
```

Comment:
no, its overcomlicated

IF version == 4 then   # for old miner version
score = 0.2 # we need to find this value by experimenting in scorer unit tests, i provided the file previously
ELSE 
   current logic